x/application/keeper/application.go

```diff
@@ -1,11 +1,11 @@
 package keeper
```

Comment:
@red-0ne Are you sure this doesn't require an update to a protocol migration to ensure the indecies are in place or default/empty proto values are in place?

Reply:
We do not need a protocol upgrade.

The `Application`s and `Supplier`s will be hydrated with empty `ServiceUsageMetrics` and will updated (and indexes created) once they are persisted.